packages/roosterjs-editor-dom/lib/table/VTable.ts

```diff
@@ -90,6 +96,10 @@ export default class VTable {
                     }
                 });
             });
+            const format = getTableFormatInfo(this.table);

```

Comment:
Suggest we read format info in the constructor of VTable. So that we can change its value directly if we want, then writeBack() will apply the change

Reply:
Since we are applying the style after we create a VTable, if we try to get the current format in the constructor, the current format will be always the default